### DISCUSSION TITLE
How best to submit an extension fix?

### DISCUSSION THREAD

**Discussion by egerlach** [chocolatey-community/chocolatey-packages#2109] [Q&A]
While working on things for 1password I came across a behaviour in the core extension that required a [quick fix](https://github.com/egerlach/chocolatey-packages/commit/37db862755497661d06a4674b251b0aaa827dddf). I have what I hope are a few quick questions regarding submission:

1. Is an issue needed or can I just create a PR?
2. Since this isn't specific to a package what if anything should I put in the prefix of my PR in parentheses?
3. Anything else I should consider?

**Comment by RedBaron2** [comment]
@egerlach 
This is already done via wormie au helpers expand-alias script

**Reply by egerlach** [reply]
Thanks for the pointer. That definitely helps with finding and expanding such things... I suppose I'm suggesting that they shouldn't be aliases in the original source, so that no one needs to worry about it ever again.

**Reply by RedBaron2** [reply]
@egerlach 
I'm sure it is just a slight oversight on the management of the repository to have the alias left in the code. It should be mentioned to @AdmiringWorm that alias are still present in a chocolatey package even a extension package.

**Reply by AdmiringWorm** [reply]
Extension packages are meant to be removed from this repository (since it requires some changes in our update scripts this hasn't been done anymore, see https://github.com/chocolatey-community/chocolatey-packages/issues/1824).

PR's to the extension packages in this repository won't be accepted, but if the aliases are still present in the new repository for extensions (https://github.com/chocolatey-community/chocolatey-extensions), then feel free to put in a PR there (it is preferred to have an issue created first).

also @RedBaron2 the expand aliases cmdlet was never run on extension packages automatically, only the packages that either explicitly called the cmdlet or was an automatic package.
